libs/@guardian/libs/src/consent-management-platform/usnat/getConsentState.ts

```diff
@@ -1,18 +1,29 @@
 import { type USNATConsentState } from '../types/usnat';
 import { getGPPData } from './api';
 
+const getSupportedAPIv1 = (
+	supportedAPI: string | undefined,
+): string | undefined => {
+	if (supportedAPI?.includes('usnat')) {
```

Comment:
This will for the majority of states but there are 5 states that have their own id's.   'usnat', 'usca','usva', 'usco', 'usut', 'usct'

Reply:
Thanks @akinsola-guardian have updated